devtools/docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+version: '3.4'
```

Comment:
Either this one or the one below shouldn't be there? Or do each have a different purpose?